ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Index.cshtml.cs

```diff
@@ -125,7 +125,8 @@ private List<CaseActionCreateHelper> GetStartHelpers()
 			{
 				new SrmaCreateHelper(_srmaService),
 				new FinancialPlanCreateHelper(_financialPlanModelService),
-				new NtiCreateHelper(_ntiUnderConsiderationModelService, _ntiWarningLetterModelService, _ntiModelService)
+				new NtiCreateHelper(_ntiUnderConsiderationModelService, _ntiWarningLetterModelService, _ntiModelService),
```

Comment:
I' a bit unsure about this. If the helpers are stateless then it shouldn't be necessary to create new ones for each list element.
And should they be dependency injected instead

Reply:
Discussed with Chris Its fine how it is